playground/frontend/lib/modules/editor/repository/code_repository/code_client/grpc_code_client.dart

```diff
@@ -34,20 +34,18 @@ import 'package:playground/utils/replace_incorrect_symbols.dart';
 const kGeneralError = 'Failed to execute code';
 
```

Comment:
Please add a comment explaining why only `runCode` uses the SDK and the rest make requests to the default API endpoint.

Reply:
Thanks @KonradJanica , I added it before `_createRunCodeClient` method

Reply:
Cool. Thanks!